gplugins/lumerical/recipes/taper_design_recipe.py

```diff
@@ -0,0 +1,389 @@
+import hashlib
+import typing
+from functools import partial
+from pathlib import Path
+
+import gdsfactory as gf
+import numpy as np
+import pandas as pd
+from gdsfactory import Component
+from gdsfactory.components.taper_cross_section import taper_cross_section
+from gdsfactory.config import logger
+from gdsfactory.pdk import LayerStack, get_layer_stack
+from gdsfactory.typings import ComponentFactory, CrossSectionSpec, PathType, WidthTypes
+from pydantic import BaseModel
+
+from gplugins.design_recipe.DesignRecipe import DesignRecipe, eval_decorator
+from gplugins.lumerical.config import cm, um
+from gplugins.lumerical.convergence_settings import (
+    LUMERICAL_EME_CONVERGENCE_SETTINGS,
+    ConvergenceSettingsLumericalEme,
+    ConvergenceSettingsLumericalFdtd,
+)
+from gplugins.lumerical.eme import LumericalEmeSimulation
+from gplugins.lumerical.recipes.fdtd_recipe import FdtdRecipe
+from gplugins.lumerical.simulation_settings import (
+    LUMERICAL_EME_SIMULATION_SETTINGS,
+    SimulationSettingsLumericalEme,
+    SimulationSettingsLumericalFdtd,
+)
+
+
+def example_run_taper_design_recipe():
```

Comment:
Great job getting this set up! One thought- `DesignRecipe`s were designed to be composed- maybe this could be refactored as an EME design recipe, an FDTD design recipe, and a `RoutingTaperDesignRecipe` with the following structure

`RoutingTaperDesignRecipe` depends on `FDTDDesignRecipe` which depends on `EMEDesignRecipe`.

Then in the example you can just do `RoutingTaperDesignRecipe.eval()` and it'll evaluate all the dependencies automatically.